### DISCUSSION TITLE
Virus?

### DISCUSSION THREAD

**Discussion by ghost** [valinet/ExplorerPatcher#2577] [General]
Somehow my Windows Updates doesnt work anymore because of explorerpatcher :(
My pc is destroyed ig..

**Comment by pyrates999** [comment]
see here for how to uninstall EP: https://github.com/valinet/ExplorerPatcher/wiki/Installer-How-To

There are currently 4 ways of doing this:

Identify "ExplorerPatcher" in Settings > Apps > Apps & features. Click the "three dots" button, and then click "Uninstall" and confirm the prompts.
Identify "ExplorerPatcher" in Control Panel > Programs and Features. With it highlighted, click "Uninstall" and confirm the prompts.
Run ep_setup.exe /uninstall.
Rename ep_setup.exe to ep_uninstall.exe and execute that.
